### DISCUSSION TITLE
Wyze Cam Pan V3 ✅

### DISCUSSION THREAD

**Discussion by petrax5** [mrlt8/docker-wyze-bridge#670] [General]
Wyze Pan V3 works as expected 

in web UI(home assistant) showing :

model_name | Pan V3
firmware_ver | 4.50.0.3405

{I have no idea how to work GitHub, I just wanted to help as the biggest thing that was holding me back from instantly buying the pan V3 was not knowing whether it would work or not with this, I hope I could help}

**Comment by mrlt8** [comment]
Thank you!! I updated the readme to include the Pan V3!
